lib/cc/analyzer/issue_location_format_validation.rb

```diff
@@ -30,13 +30,13 @@ def valid_positions?(positions)
       def valid_position?(position)
         position &&
           (
-            (position["line"] && position["column"]) ||
-            position["offset"]
+            [position["line"], position["column"]].all? { |value| value.is_a?(Integer) } ||
+            position["offset"] && position["offset"].is_a?(Integer)
```

Comment:
I see. If you're confident that this change won't cause a bunch of unexpected validation errors, then I'm totally good with the strict approach. My concern was that we had a lot of engines reporting string locations and the system has just been OK with it to this point.